scripts/powertoys/powertoys-install-context.reg

```diff
@@ -0,0 +1,14 @@
+Windows Registry Editor Version 5.00
```

Comment:
Do not add nested name.

`git mv scripts/powertoys/powertoys-install-context.reg scripts/powertoys/install-context.reg`
`git mv scripts/powertoys/powertoys-uninstall-context.reg scripts/powertoys/uninstall-context.reg`